### ISSUE TITLE
sem_seg learning rate clip typo

### ISSUE THREAD

**Issue by themmes** [charlesq34/pointnet#24]
Dear Charles,

In `sem_seg/train.py` probably a typo which eliminates your learning rate clipping, which you so emphasize.

`learing_rate` should be `learning_rate`?

```
def get_learning_rate(batch):
--
def get_learning_rate(batch):
    learning_rate = tf.train.exponential_decay(
                        BASE_LEARNING_RATE,  # Base learning rate.
                        batch * BATCH_SIZE,  # Current index into the dataset.
                        DECAY_STEP,          # Decay step.
                        DECAY_RATE,          # Decay rate.
                        staircase=True)
    learing_rate = tf.maximum(learning_rate, 0.00001) # CLIP THE LEARNING RATE!!
    return learning_rate
```

**Comment by charlesq34** [comment]
Thanks for spotting that!
I've fixed it.

**Comment by themmes** [comment]
You're welcome, did you also get a chance to look at the email I send you (charlesq34@gmail.com)?

**Comment by charlesq34** [comment]
oops, thanks again.

What's the title of the email? I cannot find any..

**Comment by themmes** [comment]
PointNet++ multi-scale (2questions, 2minread)
Sent on 6th of July to charlesq34@gmail.com

Maybe interesting to sidenote; I am currently preparing the [semantic3d.net](http://www.semantic3d.net) datasets to perform Semantic Segmentation on using your model.

**Comment by themmes** [comment]
Odd, I have send you a new email. Let me know if it does not reach you.

**Comment by belevtsoff** [comment]
@themmes how did it go with semantic3d dataset? I'm really curious how would PointNet++ perform on the large scale outdoor data.

**Comment by themmes** [comment]
@belevtsoff Thanks for asking, I discussed this with Maxim Tatarchenko (PhD from Freiburg). I discontinued the research on Semantic3D, mainly because I received another dataset (MLS, sadly not public) where I could exploit reference to the scanner. My train of thought here was that it could provide spatial reference in a similar way the PointNet on indoor exploits relative position in the room. I believe Maxim also discontinued the research on PointNet after getting results of about 17% accuracy.

**Comment by belevtsoff** [comment]
Oh, I see, thanks! Did you and Maxim do experiments with vanilla PointNet or pointnet++ as well? Seems to me like it could have a considerable improvement for large datasets.

**Comment by mtatarchenko** [comment]
Hi,

A colleague of mine and I only tried the vanilla PointNet. PointNet++ should definitely produce better results.


Best,
Maxim

**Comment by belevtsoff** [comment]
Cheers guys. I'll let you know If I'll end up trying PointNet++ on some outdoor or aerial data.

**Comment by maximiliangoettgens** [comment]
@belevtsoff did you already perform the tests on outdoor data? I am currently working on a project on outdoor data and would be very interested in an exchange

**Comment by themmes** [comment]
@belevtsoff Yes, please keep us posted. I ended up moving to proprietary data of a highway scene with PointNet (achieved about 50% MIOU for four classes excl. background class) as the PointNet++ code was not yet released at that time.

**Comment by belevtsoff** [comment]
@maximiliangoettgens @themmes Hey guys, just noticed the messages. Unfortunately, I had to drop out of the project, so I never ended up trying PointNet++ on the outdoor data. I'll forward this conversation to the guys from the project, maybe they've tried something by now.

**Comment by bw4sz** [comment]
bump on whether pointnet++ was every applied to semantic3d benchmark? Does anyone have any updates?
